packages/admin/admin/src/form/FinalFormFileSelect.tsx

```diff
@@ -143,17 +263,22 @@ export interface FinalFormFileSelectProps extends FieldRenderProps<File | File[]
     };
 }
 
-const FinalFormFileSelectComponent: React.FunctionComponent<WithStyles<typeof styles> & FinalFormFileSelectProps> = ({
-    classes,
-    disabled,
-    disableDropzone,
-    disableSelectFileButton,
-    accept,
-    maxSize = 50 * 1024 * 1024,
-    maxFiles,
-    input: { onChange, value: fieldValue, multiple: multipleFiles },
-    iconMapping = {},
-}) => {
+export function FinalFormFileSelect(inProps: FinalFormFileSelectProps) {
+    const {
+        disabled,
+        disableDropzone,
+        disableSelectFileButton,
+        accept,
+        maxSize = 50 * 1024 * 1024,
+        maxFiles,
+        input: { onChange, value: fieldValue, multiple: multipleFiles },
+        iconMapping = {},
+        ...restProps
```

Comment:
Please apply the `slotProps` to each of the slot from here: https://deploy-preview-2030--comet-dxp-docs.netlify.app/docs/comet-core-development/creating-customizable-admin-components/#slotprops-and-restprops

Reply:
https://github.com/vivid-planet/comet/pull/2072/commits/c4a9c6a23b156b029ea63f13775e57b610e03ff9